swift/vapor-framework/Sources/server/main.swift

```diff
@@ -1,6 +1,6 @@
 import Vapor
 
-var env = Environment.production
+var env = Environment(name: Environment.get("VAPOR_ENV") ?? "production")
```

Comment:
should not it be **dev** by default ? I mean the environment variable will always be set `here`, but a common usage is to set to **dev** environment (at least in `ruby`)

Reply:
Ah fair enough, done as well